backend/models/User.js

```diff
@@ -8,7 +8,7 @@ const User = new mongoose.Schema({
   lastName: { type: String, required: true },
   oauthId: { type: String, required: true, unique: true },
   propicUrl: { type: String, required: true },
```

Comment:
Let's not have `propicUrl` be required, since that would force all users to add a picture upon registration

Reply:
I agree. Good thought, @alan-fang!